.github/workflows/run-standard-tests.yml

```diff
@@ -0,0 +1,194 @@
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KNITransfer
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL_Coclustering.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KNITransfer.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KhiopsNativeInterface.dll
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/lib/libKhiopsNativeInterface.*
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/lib/KhiopsNativeInterface.lib
+          key: ${{ github.run_id }}-${{ matrix.build-setup.os }}-${{ matrix.build-setup.cmake-preset
+            }}-${{ matrix.config }}
+          fail-on-cache-miss: false
+  run-standard-tests:
+    needs: build-full-project
+    strategy:
+      fail-fast: false
+      matrix:
+        build-setup:
+          - {os: windows-2022, cmake-preset: windows-msvc}
+          - {os: ubuntu-latest, cmake-preset: linux-gcc}
+          - {os: macos-latest, cmake-preset: macos-clang}
+        running-mode: [parallel, serial]
+        config: [debug, release]
+    runs-on: ${{ matrix.build-setup.os }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+      - name: Restore cached binaries
+        id: restore-binaries
+        uses: actions/cache/restore@v3
+        with:
+          path: |
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL_Coclustering
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KNITransfer
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/MODL_Coclustering.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KNITransfer.exe
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/bin/KhiopsNativeInterface.dll
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/lib/libKhiopsNativeInterface.*
+            ${{ github.workspace }}/build/${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}/lib/KhiopsNativeInterface.lib
+          key: ${{ github.run_id }}-${{ matrix.build-setup.os }}-${{ matrix.build-setup.cmake-preset
+            }}-${{matrix.config }}
```

Comment:
Use `${{ env.PRESET_NAME }}` to shorten these lines

Reply:
done